zanata-war/src/main/java/org/zanata/rest/editor/MediaTypes.java

```diff
@@ -32,6 +32,11 @@ public String toString() {
     public static final String APPLICATION_ZANATA_LOCALES_JSON =
             APPLICATION_ZANATA_LOCALES + JSON;
 
+    public static final String APPLICATION_ZANATA_SUGGESTIONS =
+            APPLICATION_VND_ZANATA + ".suggestions";
+    public static final String APPLICATION_ZANATA_SUGGESTIONS_JSON =
+            APPLICATION_ZANATA_SUGGESTIONS + JSON;
+
```

Comment:
I think it has something to do with service auto-discovery and custom type description. Not sure if it's giving us anything the way we use it. @seanf ?